code/index.html

```diff
@@ -3,28 +3,143 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>dogGO</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://use.typekit.net/epf8jwq.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <style></style>
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
 
-    <!-- Signup form -->
+    <!--HEADER SECTION-->
+    
+    <div class="navbar-container">
+      <header class="navbar">
+        <div class="hamburger-wrapper">
```

Comment:
Could you maybe omit one of these `div`s?

Reply:
Done and done! 👍